region/norfair/west/Business Center.json

```diff
@@ -1268,142 +2551,188 @@
       ]
     },
     {
-      "link": [6, 2],
-      "name": "Leave Shinecharged Downback",
+      "link": [6, 5],
+      "name": "Come in Shinecharging, Leave Shinecharged",
       "entranceCondition": {
         "comeInShinecharging": {
-          "length": 5,
+          "length": 4,
           "openEnd": 1
         }
       },
       "requires": [
-        "canDownBack"
+        "canShinechargeMovementComplex"
       ],
       "exitCondition": {
         "leaveShinecharged": {
-          "framesRemaining": 75
+          "framesRemaining": 30
         }
       },
       "unlocksDoors": [
         {"types": ["super"], "requires": []},
         {"types": ["missiles", "powerbomb"], "requires": ["never"]}
-      ],
-      "note": "Shoot the top Sova while falling to delay its movement."
+      ]
     },
     {
-      "link": [6, 2],
-      "name": "Leave Shinecharged Kill Sova",
-      "entranceCondition": {
-        "comeInShinecharging": {
+      "link": [6, 6],
+      "name": "Leave with Runway",
+      "requires": [],
+      "exitCondition": {
+        "leaveWithRunway": {
           "length": 5,
           "openEnd": 1
         }
+      }
+    },
+    {
+      "link": [6, 7],
+      "name": "Carry Shinecharge (HiJump)",
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 100
+        }
+      },
+      "requires": [
+        "HiJump",
+        "canShinechargeMovementComplex"
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {
+          "framesRemaining": "auto"
+        }
+      }
+    },
+    {
+      "link": [6, 7],
+      "name": "Carry Shinecharge (Sova Kill)",
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 115
+        }
       },
       "requires": [
         {"enemyKill": {
-          "enemies": [["Sova"]]
-        }}
+          "enemies": [["Sova"]],
+          "explicitWeapons": ["ScrewAttack", "Wave", "Spazer", "Plasma", "Missile", "Super"]
+        }},
+        "canShinechargeMovementComplex"
       ],
       "exitCondition": {
         "leaveShinecharged": {
-          "framesRemaining": 75
+          "framesRemaining": "auto"
+        }
+      }
+    },
+    {
+      "link": [6, 7],
+      "name": "Carry Shinecharge (Sova Evade)",
```

Comment:
You can jump on the sova platform before it reaches the right edge, hehe.
93 frames was my best.

Reply:
Hmm, I assume this would have to require some run speed from the other room in order to make it in time?

Reply:
I made it without initial run speed.  A comeinshinecharging entry felt harder since you have to stop movement briefly, and more inputs.